conf/test_full.config

```diff
@@ -15,9 +15,17 @@ params {
     config_profile_description = 'Full test dataset to check pipeline function'
 
     // Input data for full size test
-    // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_full_illumina_amplicon.csv'
+    input           = 'https://raw.githubusercontent.com/sguizard/test-datasets/isoseq/samplesheet_full.csv'
+    primers         = 'https://raw.githubusercontent.com/sguizard/test-datasets/isoseq/testdata/primers_complete.fasta'
```

Comment:
Same here, add to test-dataset repo

Reply:
Done, I pushed data on a isoseq branch on test-dataset.